pyodide-build/pyodide_build/create_xbuildenv.py

```diff
@@ -13,44 +12,41 @@
 from .recipe import load_all_recipes
 
 
-def make_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
-    parser.description = (
-        "Create xbuild env.\n\n"
-        "Note: this is a private endpoint that should not be used "
-        "outside of the Pyodide Makefile."
-    )
-    return parser
-
-
-def copy_xbuild_files(xbuildenv_path: Path) -> None:
-    PYODIDE_ROOT = get_pyodide_root()
+def _copy_xbuild_files(
+    pyodide_root: Path, xbuildenv_path: Path, skip_missing_cross_file: bool = False
+) -> None:
     site_packages = Path(get_make_flag("HOSTSITEPACKAGES"))
     # Store package cross-build-files into site_packages_extras in the same tree
     # structure as they would appear in the real package.
     # In install_xbuildenv, we will use:
     # pip install -t $HOSTSITEPACKAGES -r requirements.txt
     # cp site-packages-extras $HOSTSITEPACKAGES
     site_packages_extras = xbuildenv_path / "site-packages-extras"
-    recipes = load_all_recipes(PYODIDE_ROOT / "packages")
+    recipes = load_all_recipes(pyodide_root / "packages")
     for recipe in recipes.values():
         xbuild_files = recipe.build.cross_build_files
         for path in xbuild_files:
+            source = site_packages / path
             target = site_packages_extras / path
             target.parent.mkdir(parents=True, exist_ok=True)
-            shutil.copy(site_packages / path, target)
 
+            if not source.exists():
+                if skip_missing_cross_file:
```

Comment:
This is mostly for tests. I don't want to build scipy and numpy to test this functionality